base/mqueue/platform_event.go

```diff
@@ -83,46 +85,51 @@ func batchSize(grouped groupedData) int {
 	return batches
 }
 
-func (data *InventoryAIDs) groupData() (int, *groupedData) {
+func (accs *InventoryAccounts) getAccountInventories() (int, *accountInventories, *orgIDs) {
```

Comment:
there was an intention of having the same API/function name for multiple receiver types, no strong preference tho

Reply:
I see the reason for `WriteEvents` same function name, it's part of `MessageData` interface, but I have no clue for these getters. What is the reason?

It's better to use names that suggest what variable holds. These new names may be improved, but `InventoryAccounts` is still lesser set than `groupedData`, that can refer to almost anything :smile: